src/agent/v8inspector.ts

```diff
@@ -55,10 +55,9 @@ export class V8Inspector {
   }
 
   evaluateOnCallFrame(
-      callFrameId: string, expression: string, objectGroup?: string,
-      includeCommandLineAPI?: boolean, silent?: boolean,
-      returnByValue?: boolean, generatePreview?: boolean,
-      throwOnSideEffect?: boolean) {
+      callFrameId: string, expression: string, returnByValue?: boolean,
```

Comment:
Why did the order of these parameters change?

Reply:
Since I will only use callframeId, expression and returnByValue, i will move these three together so that when I call the function, I can just pass three variables.  the remaining parameters is matched with protocal (which will not likely to use for now)

Reply:
If you have more than 3 arguments to a function, it might be worthwhile to accept a single `options` object instead, and take advantage of required and optional fields in TypeScript. This way you don't have to re-order arguments to enhance code readability.